api/Operation.md

```diff
@@ -0,0 +1,74 @@
+# Operation
+
+## Web Server Configuration
+
+Diem node generates the following default API configuration:
+
+```
+api:
+  enabled: true
+  address: "0.0.0.0:8080"
+```
+
+It's optional to configure TLS certificate and private key file paths for enabling TLS for the web server:
+```
+api:
+  enabled: true
+  address: "0.0.0.0:8080"
+  tls_cert_path: <file path>
+  tls_key_path: <file path>
+```
+
+When `api.enabled` is set to `true`, both API and JSON-RPC configured web server will serve the REST API.
+
+### JSON-RPC is enabled
+
+Regardless what is JSON-RPC `address` configuration, JSON-RPC API is always enabled at the REST API port.
+
+However, JSON-RPC service still reads configurations from `json_rpc` section.
```

Comment:
What happens after the reading of the `json_rpc` section, is JSON-RPC API available at two different addresses/ports? `json_rpc` section and `api.address` section?

Reply:
In the first section I mentioned:
```
When `api.enabled` is set to `true`, both API and JSON-RPC configured web server will serve the REST and JSON-RPC API.
```
Technically they are same web servers that serve same API (both JSON-RPC and REST), but serve at different ports if `api.address` and `json_rpc.address` are configured with different ports.
It's just we have `json_rpc` configuration, and I don't want to make a breaking change to get rid of `json_rpc.address` when we first time rollout `api`.
It's a little odd at current stage, but it is for us to migrate off json-rpc easier.